runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -151,4 +151,25 @@ public Query query(@Nonnull final String namespace) {
                 this.runtime.getObjectsView(),
                 namespace);
     }
+
+    /**
+     * Subscribe to a specific a table in a namespace or the entire namespace.
+     * Objects returned will honor transactional boundaries
+     * @param streamer - callback context.
+     * @param namespace - the CorfuStore namespace to subscribe to.
+     * @param tableName - If specified, only updates from this table will be returned.
+     * @param timestamp - If specified, all stream updates from this timestamp will be returned
+     *                  - if null, only future updates will be returned.
+     */
+    public void subscribe(@Nonnull Streamer streamer, @Nonnull String namespace,
```

Comment:
Not sure if it's a good idea to make the caller pass null.
Should we expose multiple signatures?

Reply:
without null, the number of signatures multiples quickly, that's why I just use null

Reply:
Ok.